pkg/minikube/cluster/cluster.go

```diff
@@ -340,6 +339,12 @@ func createHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 		return nil, fmt.Errorf("Error attempting to save store: %s", err)
 	}
 
+	// Create minishift-data.json
+	_, err = minishiftdata.Create(api.GetMachinesDir())
+	if err != nil {
+		return nil, fmt.Errorf("Error creating minishift-data.json: %s", err)
```

Comment:
Would like to see Create handle the actual error message creation. This should only handle the error and/or pass on the error as-is.